compiler/src/dmd/timetrace.d

```diff
@@ -23,7 +23,7 @@ import dmd.common.outbuffer;
 import dmd.root.string : toDString;
 
 // Thread local profiler instance (multithread currently not supported because compiler is single-threaded)
```

Comment:
This comment still says "thread local profiler instance"

Reply:
The compilation was failing on Windows builds because of the static. I'm not sure how to resolve this.

Reply:
I'm not sure what you mean. Can you be specific?

Reply:
The comment by the code at hand says the instance `timeTraceProfiler` of `TimeTraceProfiler` would be thread-local:
```d
// Thread local profiler instance (multithread currently not supported because compiler is single-threaded)
__gshared TimeTraceProfiler* timeTraceProfiler = null;
```
But as it is `__gshared` now, it’s no longer actually thread-local.

Reply:
Reduced to highlight the contradiction:
```d
// Thread local
__gshared T* identifier = null;
```